php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2077,7 +2077,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
             if (!in_array($key, $this->_doubleDataEntryDiffIgnoreColumns)) {
                 if ($otherData[$key] != $value) {
                     $diff[] = [
-                        'TableName'      => $this->table,
```

Comment:
I think this line is the only one that is required for the bug fix. The rest is just to make the column name "right" but involves changing the default schema, and regenerating the release patch very late in the release testing.

I think we should just change `$this->table` to `$this->testName` here for this release, and make the rest of the SQL changes (which are just renaming the column and the code to go with it) in the next release.